### PR TITLE
Adjust version formatting and add formatted version to rpc

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
see commits

### CODE REVIEW

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
Unit tests fail + see below

File: src/rpc/net.cpp [hunk 1]
```diff
@@ -432,7 +432,8 @@ UniValue getnetworkinfo(const JSONRPCRequest& request)
             "\nResult:\n"
             "{\n"
             "  \"version\": xxxxx,                      (numeric) the server version\n"
-            "  \"subversion\": \"/Dash Core:x.x.x/\",     (string) the server subversion string\n"
+            "  \"formattedversion\": \"x.x.x.x-rcx\",   (string) the formatted version including RC info or commit as relevant\n"
+            "  \"subversion\": \"/Dash Core:x.x.x/\",   (string) the server subversion string\n"
```
[the remark this answers is not in the document]
Comment (thephez) [reply src/rpc/net.cpp:436]: :+1: I like `buildversion` as well

**Comment by PastaPastaPasta** [comment]
adjusted tests

**Comment by UdjinM6** [comment]
>adjusted tests

👀 

Are you sure?

**Comment by PastaPastaPasta** [comment]
Gaaah, I hate it when I forget to push :D

**Review by UdjinM6 — APPROVED** [review]
ACK

**Review by xdustinface — APPROVED** [review]
utACK

**Pull request merged**